src/napari/_qt/widgets/qt_welcome.py

```diff
@@ -190,6 +208,10 @@ def dropEvent(self, event):
         self._update_property('drag', False)
         self.sig_dropped.emit(event)
 
+    def _next_tip(self, event=None):
+        self._tip.setText(f'Did You Know?\n{next(self._tips)}!')
+        self._tip_timer.start(5000)
```

Comment:
```suggestion
        self._tip_timer.start(10000)
```
Seemed a bit fast for me, but not a blocker.